registry/quay.io/biocontainers/diamond/container.yaml

```diff
@@ -5,9 +5,8 @@ description: Accelerated blast compatible local sequence aligner
 latest:
   2.0.14--hdcc8f71_0: sha256:275f3b3c587f8a40a39693db8acd91da8be6f053ff1426863da22061bd4e7957
 tags:
-  0.8.26--h2e03b76_4: sha256:1bdf8de97dfdfb795a14cecf22cc7b249801ac9886e187533c3b1cf3c9cb7613
-  2.0.11--hdcc8f71_0: sha256:cb3a52f7f7d745e3c023b40faebd5099afceddb290bd65ebd892ddabef41463a
-  2.0.13--hdcc8f71_0: sha256:2c70b7e6e92116163822053400f8528bae95b8846073fa61224cae14cf1ed0dc
   2.0.14--hdcc8f71_0: sha256:275f3b3c587f8a40a39693db8acd91da8be6f053ff1426863da22061bd4e7957
+  0.9.36--h56fc30b_0: sha256:0f80076288d495263598b24d8684da078a86f359bd58e39958cb491abd817399
+  0.8.36--h8b12597_4: sha256:13476bcb2a7077eae93db47fcc9f7f7662fdcb06193dc95139c7a6cf7c2de824
```

Comment:
It looks funny but that's normal. They skipped v1.* 🤷🏼 
But why are they no other 2.0.* versions ? For a given x.y stem does it only select the highest (z,t) – assuming the pattern x.y.z--*_t ?

Reply:
This is what I'm going to call "too many tag" syndrome. Because crane only gives us the top 50, we only see a sample and not any with 2.0. This is an example that would need to have them manually added. https://crane.ggcr.dev/ls/quay.io/biocontainers/diamond